shared/common-adapters/index.desktop.js

```diff
@@ -8,3 +8,4 @@ export {default as Logo} from './logo'
 export {default as Text} from './text'
 export {default as Terminal} from './terminal'
 export {default as Icon} from './icon'
+export {lineClamp} from './text'
```

Comment:
I don't think this should be here, this isn't a component.

If you need line clamp you should import it from text directly